.github/workflows/ros-build.yml

```
@@ -1,56 +1,70 @@
-on: [push, pull_request]
+on: [ push, pull_request ]
 
 jobs:
-  build_docker: # On Linux, iterates on all ROS 1 and ROS 2 distributions.
+  build_docker:
+    # On Linux, iterates on all ROS 2 distributions.
+    name: ${{ matrix.name }}
     runs-on: ubuntu-latest
     env:
       DEBIAN_FRONTEND: noninteractive
+    continue-on-error: ${{ matrix.allow_failure == true }}
```

actionlint (1.7.12)

[error] property "allow_failure" is not defined in object type {docker_image: string; name: string; ros_distribution: string; use_ros_testing: bool}

(expression)